src/lib/components/DeleteButton.js

```diff
@@ -80,6 +110,8 @@ export class DeleteButtonComponent extends Component {
 
 const mapStateToProps = (state) => ({
   isSaved: Boolean(state.deposit.record.id),
+  isPublished: state.deposit.record.is_published,
+  isNewVersion: state.deposit.record.versions?.index > 1,
```

Comment:
I got tripped up by this and maybe others might be too :sweat_smile: : `isNewVersion` -> `isVersion` could help make it clear that this value doesn't mean it's necessarily a new version, but rather that it is a (subsequent) version of a record (new only if it hasn't been published)